packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -110,7 +115,7 @@ function ScreenRoot() {
 				</ItemGroup>
 			</CardBody>
 
-			{ canEditCSS && (
+			{ canEditCSS && customCSS && (
```

Comment:
Is `customCSS` a string?

If so, I think it'd be good to coerce it to a boolean.

Reply:
Done.